pyproject.toml

```diff
@@ -14,7 +14,7 @@ description = "Hydrological Simulation Program - Python"
 dependencies = [
     "cltoolbox",
     "numba",
-    "pandas",
+    "pandas>=1.5,<2",    
```

Comment:
we should refactor all calls to `df.append()` since it's deprecated. use `pandas.concat([df1, df2])` going forward

Reply:
Because append went from warning to error with V2 and I wanted to punt the refactor. Sounds like we probably caught the same one that is in the test line-of-fire, but there may be many more lurking…